### ISSUE TITLE
Xcode 7.2 Issue

### ISSUE THREAD

**Issue by rashadz** [robbiehanson/XcodeColors#72]
After Updating to Xcode 7.2, this is not working anymore

**Comment by paradoxally** [comment]
Same here, doesn't seem to work with CocoaLumberjack with Xcode 7.2.

I used:

```swift
DDLog.addLogger(DDTTYLogger.sharedInstance())
DDTTYLogger.sharedInstance().colorsEnabled = true
```

And then doing `DDLogInfo("Info")` does not show up in green, or any of the other colors (red for error, orange for warning).

**Comment by kwade101** [comment]
Whenever there is a Xcode version change, you'll need to add the Xcode version's new UUID into the list of DVTPlugInCompatibilityUUID entries in the XcodeColors info.plist file.  I had the same issue and this resolved it.

For 7.2, the UUID is F41BD31E-2683-44B8-AE7F-5F09E919790E.

**Comment by paradoxally** [comment]
@kwade101 No, that's not it. The latest version of this plugin already had the 7.2 UUID in it and Xcode loads it.

**Comment by kwade101** [comment]
@paradoxally Ah, ok. My apologies. All I saw was the mention of an update to Xcode, not to the plugin.

**Comment by robbiehanson** [comment]
Have you re-installed XcodeColors ?

- Checkout the latest via git
- Build-and-go on project
- Restart Xcode
- Build-and-go again (should see colors in console)

Another thing that may help (I've heard rumors of it helping)

Try installing plugin via Alcatraz:
http://alcatraz.io/

Let me know what you find. I'd like to write up a wikipage for troubleshooting these issues.

**Comment by paradoxally** [comment]
@robbiehanson Yes, I tried reinstalling and I'm using Alcatraz. The bundle is loaded by Xcode, but it simply won't show colors using CocoaLumberjack.

**Comment by robbiehanson** [comment]
If you run the Xcode project within the XcodeColors repository, does it work then ?

**Comment by paradoxally** [comment]
@robbiehanson Yes, the test project works.

I found the fix from [here](https://github.com/CocoaLumberjack/CocoaLumberjack/issues/50#issuecomment-34286656) though. It turns out that an environment variable named `XcodeColors` with value `YES` must be added to the current scheme (mine is `Run > Debug`, but you can set it for any of them).

After adding that env. variable, it now logs with colors using CocoaLumberjack.

**Comment by rashadz** [comment]
installing via http://alcatraz.io/ worked for me:)

**Comment by Skornos** [comment]
Had the same problem and it was solved as @paradoxally suggested

**Comment by mamaral** [comment]
@robbiehanson's solution worked for me. Thanks!

**Comment by galambalazs** [comment]
Installing from Alcatraz, but still needed the flag mentioned by @paradoxally

**Comment by zetasq** [comment]
+1

**Comment by rsattar** [comment]
The issue with requiring the `XcodeColors` environment variable is that it's going to be set to YES even on people's computers that don't have XcodeColors installed, and then you'll see the inserted color characters in your logs. So it seems that for some reason in Xcode 7.2+ (7.3.1 in my case), the XcodeColors plugin isn't able to set the `XcodeColors=YES` environment variable so that, we can make a runtime decision on whether to enable colors or not.
